### PR TITLE
add  solution

### PR DESCRIPTION
**Pull request by SSDoizha**
- [X] Header height is set in 1 place (for the links)
- [X] Content is vertically centered (for any header height)
- [X] CSS is used to show all letters in Uppercase (don't type them in HTML)
- [X] Logo is an image wrapped with a link
- [X] **CSS Variable** is used for a blue color
- [X] Pseudo-element is used for a blue line below the active link
- [X] Code follows all the [Code Style Rules ❗️](./checklist.md)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,77 @@
...
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px 0;
+}
+
+.header {
+  width: 100%;
+  background-color: #ffff;
```
Comment (anastasiiavorobiova) [inline src/style.css:20]:
```suggestion
  background-color: #fff;
```

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,77 @@
...
+.is-active,
+.nav_link:hover {
+  color: var(--color-nav-link);
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  position: absolute;
+  width: 37px;
```
Comment (anastasiiavorobiova) [inline src/style.css:72]:
```suggestion
  width: 100%;
```

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,77 @@
...
+  height: 4px;
+  top: 56px;
```
Comment (anastasiiavorobiova) [inline src/style.css:74]:
```suggestion
  bottom: 0;
```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done!

**Review by lerastarynets — APPROVED** [review]
Keep up the great work!

**Pull request closed without merging**